Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.yml

```diff
@@ -61,6 +61,14 @@ configuration:
   name: proxy
   type: 8
   required: false
+- display: E-Mail
+  name: email
+  required: false
+  type: 0
+- display: Company Name
```

Comment:
Please mention in both parameters that they will be used for analytics so that the customers will be aware of that. When it's done we are good to go

Reply:
I have made the changes @reutshal . Please verify and let me know if there are any more changes needed to be done on our end.